testcases.py

```diff
@@ -1322,6 +1334,77 @@ def check(self) -> TestResult:
         return TestResult.SUCCEEDED
 
 
+class TestCaseConnectionMigration(TestCaseAddressRebinding):
+    @staticmethod
+    def name():
+        return "connectionmigration"
+
+    @staticmethod
+    def abbreviation():
+        return "CM"
+
+    @staticmethod
+    def testname(p: Perspective):
+        if p is Perspective.CLIENT:
+            return "connectionmigration"
+        return "transfer"
+
+    @staticmethod
+    def desc():
+        return "A transfer succeeded during which the client performed an active migration."
+
+    @staticmethod
+    def scenario() -> str:
+        return super(TestCaseTransfer, TestCaseTransfer).scenario()
+
+    def get_paths(self):
+        self._files = [
+            self._generate_random_file(2 * MB),
+        ]
+        return self._files
+
+    def check(self) -> TestResult:
+        result = super(TestCaseConnectionMigration, self).check()
```

Comment:
Can you add a comment here, something like "The parent check() method ensures that the client changed addresses and that PATH_CHALLENGE/RESPONSE frames were sent and received."